lago/prefix.py

```diff
@@ -1225,6 +1226,7 @@ def _deploy_host(self, host):
                 return
 
             with LogTask('Wait for ssh connectivity'):
+                time.sleep(5)
```

Comment:
Well, the issue is that the deploy command is not used only on boot, but also whenever you want no? It's a command of it's own.

Sorry I might not have explained well, let me rephrase, we should do one of:
   * Instead of doing the sleep first, do a first try, and then sleep (if keeping it in the deploy step).
   * Move the sleep to wherever we do the boot (probably the start step).